less/home-page.less

```diff
@@ -26,6 +41,12 @@
   .inverse-content {
     padding-bottom: 30px;
     border-bottom: 2px dashed @navigation-border;
```

Comment:
Using `@navigation-border` here can be a bit misleading since what you're actually setting here is the colour. Since it is shared between all borders on the page, you could simply rename it to `@border-color` which would make more sense to other developers who read it.